examples/fault-injection/docker-compose.yaml

```diff
@@ -13,6 +13,8 @@ services:
     ports:
       - 9211:9211
       - 9901:9901
+    environment:
+      ENVOY_UID: 0
```

Comment:
why is this necessary?

Reply:
FWIW https://github.com/envoyproxy/envoy/pull/11523

Reply:
It was necessary to give Envoy permission to access `/dev/stdout`. Without that setting Envoy container exists complaining that it cannot write logs to `/dev/stdout` (specified in envoy.yaml as sink for logs).

Reply:
Optional, bonus points for commenting that - I ran into that trying to repro and it'd be lovely if a local grep for stdout turned up the fix :-)

Reply:
Added comment to explain why Envoy must run as root.